service/frontend/handler.go

```diff
@@ -651,8 +651,8 @@ func (wh *WorkflowHandler) GetWorkflowExecutionHistory(
 			Execution:  getRequest.GetExecution(),
```

Comment:
This comment is intended for line 643:
if IsSetNextPageToken() 
   - Can we verify the pagetoken - run ID is valid UUID like above and that the run ID matches with runID on the request.